### ISSUE TITLE
Installation notes required

### ISSUE THREAD

**Issue by lezhnev74** [RyuuGan/sol-merger#3]
Hi! Noticed your package from this post: https://github.com/trufflesuite/truffle/issues/564 
I am not familiar with Node.js but I want to use the tool :)
Can you summarize how to install it?

Thanks!

**Comment by lezhnev74** [comment]
Found this way: `node bin/sol-merger.js /path/to/contract.sol`

**Comment by RyuuGan** [comment]
Hello,

This is one of the way of using this package. The most common way is described by `simple usage`.  Let me describe the common way.

After installing the `Node.js` you have a tool called `npm` (node package manager) in your path. So perform next steps at terminal:

1. First you need to initialize you project with `npm init` command. Follow the intructions, by that command. So you will initialize and `npm` project and file with name `package.json` will appear.

2. Execute `npm i --save-dev sol-merger` command in a directory, where you initialized project.

3. Add following lines to `package.json` 
    ```
    "scripts": {
        "build-contracts": "sol-merger './contracts/*.sol' ./build"
     }
    ```
   Note if `scripts` does not exist add it, if it is exists just add `"build-contracts"` line under the `scripts`.

4. Execute `npm run build-contracts` to build all contracts.

---

Note that `npm ...` commands should be executed in directory where npm project is inited (`package.json` file presents).

Best regards,
Valery Aligorsky.
